docs/pages/careers.tsx

```diff
@@ -34,6 +34,12 @@ const openRolesData = [
           'You will drive the development and maintenance of the documentation platform that powers all MUI products.',
         url: '/careers/react-engineer-docs-infra/',
       },
+      {
+        title: 'Staff Engineer - Pigment CSS',
+        description:
+          'Research, build, document, and ship high-quality zero-runtime CSS-in-JS library with a focus on performance and great developer experience.',
```

Comment:
```suggestion
          'Research, build, document, and help ship a next-gen zero-runtime CSS-in-JS library with a focus on performance and great developer experience.',
```

Using "next-gen" to resonate with the blog post!